campaign/snowplow.js

```diff
@@ -41,16 +41,21 @@ const track = (data, action) => {
 
   const ssoGuid = data['sso_guid']
   const grMasterPersonId = data['gr_master_person_id']
-  const acs_email = data['acs_email']
+  const acsEmail = data['acs_email']
 
   const idData = { gr_master_person_id: grMasterPersonId }
+  const acsData = { acs_label: data['service_label'] ? data['service_label'] : data['delivery_label'] }
 
   if (ssoGuid) {
     idData.sso_guid = ssoGuid
   }
 
-  if (acs_email) {
-    idData.acs_email = acs_email
+  if (acsEmail) {
+    idData.acs_email = acsEmail
```

Comment:
Should this be `idData['acs_email']`? Can't remember if underscores work for attribute names consistently.

Reply:
It seems to work this way but I agree, we should do it one way or another.

Reply:
I'm actually going to go ahead and change it to be this in a separate commit.